source/options.tsx

```diff
@@ -53,8 +54,8 @@ async function getTokenScopes(personalToken: string): Promise<string[]> {
 	});
 
 	if (!response.ok) {
-		const details = await response.json();
-		throw new Error(details.message);
+		const details: JsonObject = await response.json();
+		throw new Error(String(details.message));
```

Comment:
Does this fix a specific error? Seems unnecessary if we're just going to force it into a string

Reply:
I did not mean to commit this